addons/website_event/static/src/js/website_event.js

```diff
@@ -12,13 +11,18 @@ var EventRegistrationForm = publicWidget.Widget.extend({
      */
     start: function () {
         var self = this;
+        var post = {};
+        $('#registration_form select').each(function () {
+            post[$(this).attr('name')] = $(this).val();
+        });
```

Comment:
_Suggestion:_ I see that we retrieve the `post` variable here and also in the `onclick`.
=> Maybe we could add a small private method to prevent duplication.
Something like that just under the `start` function :
```js
    _getPost: function (){
        var post = {};
        $('#registration_form select').each(function () {
            post[$(this).attr('name')] = $(this).val();
        });
        return post;
    },
```
You can then just call it this way in the `start` and the `onclick`:
```js
var post = this._getPost();
```

Reply:
Hello @amdi-odoo 🙂

Suggested changes have been made.

Thanks!